src/routes.json

```diff
@@ -247,33 +258,40 @@
         "title": "Scratch Terms of Use"
     },
     {
+        "name": "things-to-try",
+        "pattern": "^/go/?(\\?.*)?$",
+        "routeAlias": "/go/?\\??",
+        "redirect": "/tips"
+    },
```

Comment:
+1 for putting them into the redirect section, and I'm cool with renaming if that's helpful to include `-redirect` but am also fine with keeping them as they are too.